homeassistant/components/enigma2/media_player.py

```diff
@@ -0,0 +1,221 @@
+"""Support for Enigma2 media players."""
+import logging
+import asyncio
+import voluptuous as vol
+
+from homeassistant.components.media_player import MediaPlayerDevice
+from homeassistant.helpers.config_validation import (PLATFORM_SCHEMA)
+from homeassistant.components.media_player.const import (
+    SUPPORT_NEXT_TRACK, SUPPORT_PAUSE, SUPPORT_PREVIOUS_TRACK, SUPPORT_TURN_ON,
+    SUPPORT_TURN_OFF, SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, SUPPORT_STOP,
+    SUPPORT_SELECT_SOURCE, SUPPORT_VOLUME_STEP, MEDIA_TYPE_TVSHOW)
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, CONF_USERNAME, CONF_PASSWORD, CONF_SSL,
+    STATE_OFF, STATE_ON, STATE_PLAYING, CONF_PORT)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['openwebifpy==1.2.6']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_MEDIA_CURRENTLY_RECORDING = 'media_currently_recording'
+ATTR_MEDIA_DESCRIPTION = 'media_description'
+ATTR_MEDIA_END_TIME = 'media_end_time'
+ATTR_MEDIA_START_TIME = 'media_start_time'
+
+CONF_PREFER_PICON = "prefer_picon"
```

Comment:
What is the meaning of picon, program icon? Could this be worded to avoid shortening for clarity?

Reply:
Picon is the channel icon. It is a standard term in the world of Enigma2 based satellite recievers.
e.g.
![channel5hd](https://user-images.githubusercontent.com/254309/53949992-16fd2500-40c3-11e9-8920-efd941ba6bf8.png)

https://www.google.com/search?client=safari&rls=en&ei=ovCAXM6VHcPZwAKtrb_wDA&q=site%3Alinuxsat-support.com+picon&oq=site%3Alinuxsat-support.com+picon&gs_l=psy-ab.3...10297.16120..16323...1.0..0.51.1028.22......0....1..gws-wiz.......0i71.yjBhy8-fVPg

Anyone with one of these boxes will be used to the term. It isn't really an abbreviation, as I have never once seen anyone expand it longer haha.

Reply:
Would `use_channel_icon` be clearer for the users, then? :-) It's fine for me though as it is.

Reply:
Yes I like your idea here.

Reply:
Thanks @rytilahti I have changed that to `use_channel_icon` now (and I have updated the docs PR https://github.com/home-assistant/home-assistant.io/pull/8776/files#diff-5bc10a045f7eb42ea757c0ca9d0d9b17 )